examples/with-supertokens/pages/_app.js

```diff
@@ -0,0 +1,35 @@
+import '../styles/globals.css'
+import React from 'react'
+import 'regenerator-runtime/runtime'
```

Comment:
What's this for?

Reply:
This is legacy now and was removed in this PR. It had to do with babel requiring the runtime package to be present.